pkg/wfc3/calwf3/calwf3/wf3member.c

```diff
@@ -43,10 +42,9 @@ WF3Info *wf3		o: exposure specific flags and info
 
 	/* Initialize local strings */
 	rootname[0] = '\0';
-	outroot[0]  = '\0';
 	mtype[0]    = '\0';
 
-	strcpy (outroot, asn->product[prodid].subprod[posid].name);
+	const char * outroot = asn->product[prodid].subprod[posid].name;
```

Comment:
BTW I'm __not__ saying that we don't wait for Megan to review it also, just that you should be able to review this and that your comment:
> If you thought about it, then it's fine by me.

leads me to believe that you haven't been able to review it and therefore makes me question what you have reviewed on the ACS side of changes.